lib/components/Atom/Calendar/date-picker.scss

```diff
@@ -20,12 +20,44 @@
             margin-bottom: var(--semantic-spacing-general-md);
 
             button:enabled:hover,
-            button:enabled:focus {
+            button:enabled:focus,
+            button:disabled {
+                background-color: unset;
+            }
+
+            button:enabled:focus:is(.react-calendar__navigation__arrow),
```

Comment:
Unfortunately, in the library this style were written with tag selectors, so in order to overwrite them I was forced to do the same